api-report/sequence.api.md

```diff
@@ -383,7 +382,7 @@ export abstract class SequenceEvent<TOperation extends MergeTreeDeltaOperationTy
 
 // @public (undocumented)
 export class SequenceInterval implements ISerializableInterval {
-    constructor(start: LocalReference, end: LocalReference, intervalType: IntervalType, props?: PropertySet);
+    constructor(client: Client, start: LocalReferencePosition, end: LocalReferencePosition, intervalType: IntervalType, props?: PropertySet);
```

Comment:
Does this constructor signature change (specifically the new parameter) need to be called out in the breaking changes?

Reply:
this class is internal, and should not be used by anyone